lesson03/assignment/data/data.json

```diff
@@ -51,4 +51,31 @@
        "HTML", "CSS", "JavaScript", "Java", "Spring"
     ]
   }
- ]
\ No newline at end of file
+ ];
+
+const employee-info-all = template-hook.map(function(emp) {
+  let template = `
+  <section class="template-hook">
+  <div class="card">
+  <div class="left-content-card">
+  <img src="img/unsplash-headshot.jpg" alt="male avatar picture for employee id">
+
+  <h2>${emp.name}</h2>
+  <p>${emp.jobTitle}</p>
+  </div>
+  <div class="card-content">
+  <p><b>Company:</b>${emp.company}</p>
+  <p><b>Experience:</b>${emp.experience}</p>
+  <p><b>School:</b>${emp.school}</p>
+  <p><b>Major:</b>${emp.major}</p>
+  <p><b>Email:</b>${emp.email}</p>
+  <div class="linkedin-img-logo"><img src="img/linkedin.svg" alt="linkedin-logo">
+  <span class="linkedin-email">${emp.linkedInUrl}</span></div>
+
+  </div>
+  </div>
+  </section>
+  `;
+  });
+
+  $(".template-hook").append(employee-info-all);
```

Comment:
There shouldn't be any javascript in this file, it's just for holding the JSON data.